.circleci/config.yml

```diff
@@ -544,6 +544,22 @@ jobs:
           name: Publish package
           command: npm publish src/frontend
 
+  # ---- Documentation website jobs ----
+  # Deploy the docs website to GitHub pages.
+  deploy-website:
+    docker:
+      - image: circleci/node:10
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - run:
+          name: Deploying to GitHub Pages
+          command: |
+            git config --global user.email "funmoocbot@users.noreply.github.com"
```

Comment:
or should we could commit with `fun.dev@fun-mooc.fr`? WDYT @jmaupetit

Reply:
ok. 
Have you tried using the --author option directly on the commit command?

Reply:
I'm not sure how I could do this since the commit is done by the docusaurus script and not directly by us